### PR TITLE
Make vscode-apielements almost usable

### PR DESCRIPTION
**Pull request by XVincentX**
This PR will make this extension worth to be deployed and announced.

Looking forward!

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -104,29 +112,89 @@ function validateTextDocument(textDocument: TextDocument): void {
...
+      symbolType: SymbolKind.Method
+  }
+  ].forEach(({query, symbolType}) => {
+    const queryResults = refractUtils.query(refractOutput, query);
+
+    symbolArray.push(...lodash.map(queryResults, (queryResult) => {
+      let sourceMap = lodash.get(queryResult, 'attributes.sourceMap',
+        lodash.get(queryResult, 'meta.title.attributes.sourceMap',
+          lodash.get(queryResult, 'attributes.href.attributes.sourceMap',
+            lodash.get(queryResult, 'content[0].attributes.method.attributes.sourceMap')
```
Comment (smizell) [inline server/src/server.ts:177]: Just a note, in looking at this, I think that all four of these `get` functions will be executed each time.

Comment (XVincentX) [inline server/src/server.ts:177]: That's right.
Consider this as a wonky method...which works.

The idea would then be to modify drafter to include sourcemaps where missing or use again `refract-query` to get the nearest source map reference.

Does that make sense?

Comment (XVincentX) [inline server/src/server.ts:177]: @smizell
This issue is now tracked here: #3

Comment (smizell) [inline server/src/server.ts:177]: Cool 👍

**Pull request merged**